### PR TITLE
Cleanup regarding manifests and deprecated builds.

### LINKED ISSUE #271: rpi3_stable.xml not available
rpi3_stable.xml is stated as available, but it's not:

`repo init -u https://github.com/OP-TEE/manifest.git -m rpi3_stable.xml`

`fatal: manifest 'rpi3_stable.xml' not available
fatal: manifest rpi3_stable.xml not found`

### PR DESCRIPTION
**Pull request by jbech-linaro**
- Clarify how to get older releases.
- Remove references to deprecated builds like HiKey Debian (not the old stable manifest) and MTK8173.
- We're not using Travis to build matrix any more, so the icon telling the status has been removed.

This should fix #271,  #278 and partly #296.

### CODE REVIEW

**Comment by jbech-linaro** [comment]
Note to myself, add reference to the HiKey 960 ...

File: README.md [hunk 1]
```diff
@@ -83,8 +83,8 @@ information.
 | [ARM Juno Board](http://www.arm.com/products/tools/development-boards/versatile-express/juno-arm-development-platform.php) |`PLATFORM=vexpress-juno`| Yes |
 | [ARM Foundation FVP](http://www.arm.com/fvp) |`PLATFORM=vexpress-fvp`| Yes |
 | [HiKey Board (HiSilicon Kirin 620)](https://www.96boards.org/products/hikey)|`PLATFORM=hikey`| Yes |
-| [MediaTek MT8173 EVB Board](http://www.mediatek.com/en/products/mobile-communications/tablet/mt8173)|`PLATFORM=mediatek-mt8173`| No |
 | [Poplar](https://www.96boards.org/product/poplar/)|`PLATFORM=poplar`| Yes |
+| [MediaTek MT8173 EVB Board](http://www.mediatek.com/en/products/mobile-communications/tablet/mt8173)|`PLATFORM=mediatek-mt8173`| No |
```
Comment (jforissier) [inline README.md]: Unrelated

**Comment by jbech-linaro** [comment]
Squashed, rebased and tag(s) applied (should be) ready for merge! Thanks!
